instrumentation/kamon-system-metrics/src/main/scala/kamon/instrumentation/system/jvm/JvmMetrics.scala

```diff
@@ -124,6 +124,22 @@ object JvmMetrics {
     description = "Total number od classes currently loaded"
   )
 
+  val BufferPoolCount = Kamon.gauge(
+    name = s"jvm.memory.buffer-pool.count",
+    description = "Estimated number of buffers in the pool"
+  )
+  val BufferPoolUsage = Kamon.gauge(
+    name = s"jvm.memory.buffer-pool.usage",
```

Comment:
This should be changed to `used` to be in line with other similar metrics. I would also recommend add a proper `usage` metric that tracks the percentage of usage (capacity vs actual use).

Reply:
Renamed. 

Making a `usage` metric doesn't make sense, since the `BufferPoolMXBean` can only tell us the total capacity of the pool, and the amount currently used by the JVM.

Docs for `getMemoryUsed`: 
```
Returns an estimate of the memory that the Java virtual machine is using for this buffer pool. 
The value returned by this method may differ from the estimate of the total capacity of the buffers in this pool. 
This difference is explained by alignment, memory allocator, and other implementation specific reasons.
```